vendor/modules.txt

```diff
@@ -3,10 +3,13 @@ github.com/beorn7/perks/quantile
 # github.com/blang/semver v3.5.0+incompatible
 github.com/blang/semver
 # github.com/container-storage-interface/spec v1.2.0
+## explicit
```

Comment:
You must use golang 1.13 to generate vendor. And why is there any vendor change at all? There is no new import in the code.

Reply:
Yes, fixed it.